Cannot find module 'xxxx' when using path mapping

- [x] I have searched through existing issues
    I find #545 and #423, but error is still exists.
- [x] I have read through [docs](https://vuejs.github.io/vetur)
- [x] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)

## Info

- Platform: Win
- Vetur version : 0.11.7
- VS Code version:1.22.1

## Problem

![default](https://user-images.githubusercontent.com/1052632/38764219-839ecd9a-3fdd-11e8-8870-2e4c3006c6a1.png)

## Reproducible Case

npm install -g @vue/cli
vue create explorer
[example.zip](https://github.com/vuejs/vetur/files/1910017/example.zip)

The folder `client` contains a vue project created by vue-cli
The folder `empty` contains only a empty tsconfig.json

works fine:
![image](https://user-images.githubusercontent.com/1363137/60805792-c18d6d80-a1b3-11e9-87a5-60334b6aa4e1.png)


Cannot find module '@/components/HelloWorld.vue':
![image](https://user-images.githubusercontent.com/1363137/60805743-999e0a00-a1b3-11e9-8d1c-ad0b2bb609b5.png)

The only difference is order

I'm also having the same issue, and haven't found a workaround yet.

`tsconfig`:
```
{
  "compilerOptions": {
    "paths": {
      "@/*": [
        "src/*"
      ]
    }
  }
}
```
`app.vue`:
```
<script lang="ts">
import * as MyStore from '@/store/mystore'
```
Yields error
`Cannot find module '@/store/mystore'. Vetur(2307)`

If anyone finds a fix for this, I'd be super appreciative.

> The folder `client` contains a vue project created by vue-cli
> The folder `empty` contains only a empty tsconfig.json
> 
> works fine:
> ![image](https://user-images.githubusercontent.com/1363137/60805792-c18d6d80-a1b3-11e9-87a5-60334b6aa4e1.png)
> 
> Cannot find module '@/components/HelloWorld.vue':
> ![image](https://user-images.githubusercontent.com/1363137/60805743-999e0a00-a1b3-11e9-8d1c-ad0b2bb609b5.png)
> 
> The only difference is order
> 
> gif:
> ![gif5新文件 (1)](https://user-images.githubusercontent.com/1363137/60807630-c30d6480-a1b8-11e9-908d-4667ba8401f3.gif)

Very effective, solved my problem

I have the same problem, but with node_modules package. The package is installed, the code is working, but vetur throws error `Cannot find module 'vue-flickity'.Vetur(2307)`

I also get `Cannot find module '@/use/use-boards'. Vetur(2307)` error. 

boards.vue:
![image](https://user-images.githubusercontent.com/4997825/86818894-9f148580-c08f-11ea-8a2d-4d6bb307a0b1.png)

tsconfig:
```
"paths": {
      "@/*": ["src/*"]
 }
```
![image](https://user-images.githubusercontent.com/4997825/86819187-f87cb480-c08f-11ea-839e-8ca39f3cf6a0.png)

the folder:

![image](https://user-images.githubusercontent.com/4997825/86819371-37126f00-c090-11ea-9f7c-838eb2995939.png)

**Possible solution:** read the tsconfig.json that closest to the file (not in the root).

> > Any ideas ?
> 
> [#762 (comment)](https://github.com/vuejs/vetur/issues/762#issuecomment-714200659)

This solution not working, I have `tsconfig.json` with: 
```
...
    "paths": {
      "@/*": [
        "src/*"
      ]
    },
...
```
and still I have an error:
_Cannot find module '@/components/table/TableComponent.vue' or its corresponding type declarations.Vetur(2307)_

> > > Any ideas ?
> > 
> > 
> > [#762 (comment)](https://github.com/vuejs/vetur/issues/762#issuecomment-714200659)
> 
> This solution not working, I have `tsconfig.json` with:
> 
> ```
> ...
>     "paths": {
>       "@/*": [
>         "src/*"
>       ]
>     },
> ...
> ```
> 
> and still I have an error:
> _Cannot find module '@/components/table/TableComponent.vue' or its corresponding type declarations.Vetur(2307)_

if you set `baseUrl`?
You can provide a reproducible project.

--- a/server/src/services/typescriptService/serviceHost.ts
+++ b/server/src/services/typescriptService/serviceHost.ts
@@ -306,7 +306,7 @@ export function getServiceHost(
             return cachedResolvedModule;
           }
 
-          if (path.isAbsolute(name) || !isVueFile(name)) {
+          if (!isVueFile(name)) {
             const tsResolvedModule = tsModule.resolveModuleName(name, containingFile, options, tsModule.sys)
               .resolvedModule;
 

